summarization/text_summarization.py

```diff
@@ -32,12 +35,19 @@ def __init__(self, priority):
         '''
         self.text_summarizer = None
         if priority == "accuracy":
-            self.text_summarizer = Summarizer(model='distilbert-base-uncased')
+            handler = CoreferenceHandler(greedyness=.4)
```

Comment:
Could you add links to the summarizers being used (so it is easier for the reader to navigate to the library)?

Reply:
Updated accordingly!